sql/migrations/warehouse/000001_create_tables.up.sql

```diff
@@ -0,0 +1,139 @@
+
+--
+-- wh_staging_files
+--
+
+DO $$ BEGIN
+    DROP TYPE wh_staging_state_type;
```

Comment:
This should be after `ALTER TABLE wh_staging_files ALTER COLUMN status TYPE VARCHAR(255);` ?